### ISSUE TITLE
`deis logs` doesn't show log with Procfile based deployment

### ISSUE THREAD

**Issue by olalonde** [deis/deis#3967]
I recently switched from Dockerfile deployment to a Procfile based one. `deis logs` now only shows `deis[api]` logs but doesn't show any web logs. For example:

```
2015-07-01T14:36:05UTC deis[api]: olalonde scaled containers web=1 daemon=1
2015-07-06T12:29:11UTC deis[api]: olalonde deployed 72a8de3
```

Is it possible to get logs for my web and daemon processes? Viewing the logs with fleetctl works as expected.

deis v1.6.1

**Comment by bacongobbler** [comment]
Is logspout and deis-logger working on those hosts? If the logs are showing up in fleetctl then they should be carried over to the logger, as long as everything is working.

**Comment by olalonde** [comment]
Yes, all 3 logspout services are active/running and the logger service is also active/running.

**Comment by bacongobbler** [comment]
it may be worth checking the actual logs to confirm with `fleetctl ssh myapp_v2_.web.1 docker logs deis-logspout && deisctl journal logger`. We've seen cases where etcd may be unreachable and the containers won't reboot.

**Comment by olalonde** [comment]
Here's the logs of `fleetctl ssh testnet_v14.web.1 docker logs deis-logspout`

```
[...]
2015/07/01 14:35:07 event: 751db8ae534d stop
2015/07/01 14:36:16 event: 10.21.1.251: pull
2015/07/01 14:36:16 event: 62498088a4bd create
2015/07/01 14:36:17 event: 62498088a4bd start
2015/07/01 14:36:17 attach: 62498088a4bd testnet_v13.web.1 success
2015/07/01 16:06:04 event: 704fe7794d2d die
2015/07/01 16:06:04 event: 704fe7794d2d destroy
2015/07/06 12:29:34 event: 10.21.1.251: pull
2015/07/06 12:29:35 event: 93ed3dd0cb5c create
2015/07/06 12:29:36 event: 93ed3dd0cb5c start
2015/07/06 12:29:36 attach: 93ed3dd0cb5c testnet_v14.web.1 success
2015/07/06 12:29:52 attach: 62498088a4bd finished
2015/07/06 12:29:52 event: 62498088a4bd die
2015/07/06 12:29:53 event: 62498088a4bd stop
```

`deisctl journal logger`

```
[...]
Jul 06 17:29:33 ip-10-21-1-252.ec2.internal sh[30638]: 2015/07/06 17:29:33 501: All the given peers are not reachable (Tried to connect to each peer twice and failed) [0]
Jul 06 17:29:34 ip-10-21-1-252.ec2.internal sh[30638]: 2015/07/06 17:29:34 501: All the given peers are not reachable (Tried to connect to each peer twice and failed) [0]
Jul 06 17:29:35 ip-10-21-1-252.ec2.internal sh[30638]: 2015/07/06 17:29:35 501: All the given peers are not reachable (Tried to connect to each peer twice and failed) [0]
Jul 06 17:29:36 ip-10-21-1-252.ec2.internal sh[30638]: 2015/07/06 17:29:36 501: All the given peers are not reachable (Tried to connect to each peer twice and failed) [0]
```

**Comment by olalonde** [comment]
`deisctl restart logger` fixed the issue though I'm still wondering what went wrong.

**Comment by carmstrong** [comment]
> though I'm still wondering what went wrong.

That error indicates that logger was receiving errors from etcd on that host... does `journalctl -fu etcd` show anything? I think the real fix here is to use etcd2: #3564

**Comment by olalonde** [comment]
Ok thanks.
